nrf_modem/doc/dectphy.rst

```diff
@@ -0,0 +1,296 @@
+
+When a radio operation is being executed, the stack is in one of the following states:
+
+* TX active: the radio is transmitting, no reception is ongoing.
+* RX active: the radio is receiving, no transmission is ongoing.
+
+When no radio operations is being executed, the stack is in the following state:
+
+* Radio Idle: the radio is powered on and able to start a radio operation with relatively low latency, at the cost of increased power consumption compared sleep or off states.
+
+Currently, the DECT PHY layer in the DECT NR+ modem firmware does not support deep sleep states.
+
+Scheduling operations
+=====================
+
+The DECT PHY interface allows to schedule radio operations for execution by the scheduler of the DECT NR+ physical layer in the DECT NR+ modem firmware.
+Due to the nature of a radio scheduler, which allows radio operations to be executed at a specific time in the future, all radio operations in the DECT PHY interface are asynchronous and their completion is signalled to the application using callbacks.
+
+All scheduling is final, meaning that it is not possible to unschedule a scheduled operation.
+
+A radio operation may be scheduled to execute a specific time in the future, or immediately, if the radio is not currently executing any other operation.
+
+All radio operations have an application-defined handle which can be used to identify the operation and that is returned by the callback signalling the completion of an operation.
+The operation handle is entirely opaque to all underlying layers, including the DECT PHY interface.
+
+Modem time and operation latency
+--------------------------------
+
+Operation execution is scheduled by the application according to the modem time, which is a 64 bit counter kept by the modem.
+All radio operation callbacks provide the value that the modem time counter had at the moment the callback was sent by the modem core to the application core, thus providing a way for the application to track the modem time without explicitly querying the modem for it.
+
+If necessary, the application can retrieve the modem time counter value by calling the :c:func:`nrf_modem_dect_phy_time_get` function.
+
+Both radio state transitions between operations in the modem core and the inter-processor communication (IPC) mechanism between the application and more cores have latency.
+The latency of both radio state transitions and IPC is measured and provided by the DECT PHY interface upon initialization, by the :c:member:`nrf_modem_dect_phy_callbacks.init` callback function, in the :c:member:`nrf_modem_dect_phy_modem_cfg.latency` parameter.
+
+.. note::
+   The application must account for operations' latency when scheduling operations.
+
+Temperature monitoring
+======================
+
+The DECT PHY interface reports the current SoC temperature as measured by the DECT NR+ modem firmware in all radio operation callbacks, and upon initialization.
```

Comment:
Should this be SoC or SiP as we say SiP in case of the nRF91 Series devices?

Reply:
@lemrey, @divipillai : I got an clarification to this issue from RF guys: "The acronym SiP is more accurate in the context of temperature measurement as the temperature sensor locates outside of SoC but inside the SiP-module. Hence it measures the internal temperature of the SiP-module." The temperature reading as such is affected by the ambient temperature as well as the the ongoing procedure modem is executing, like TX.